content/swagger/resources/app_catalogs/routes/app_catalogs_id_product_uuids.yaml

```diff
@@ -1,7 +1,11 @@
 get:
   summary: Get the list of product uuids
   operationId: "get_app_catalog_product_uuids"
-  description: Coming soon. This endpoint allows you to get the list of uuids of products contained in a catalog.
+  description: |
+    Coming soon. This endpoint allows you to get the list of uuids of products contained in a catalog.
+    Note that a disabled catalog can return an empty list.
```

Comment:
Is it true? I had understood that we will display an error message in that case: "No products to synchronize. The catalog 2983dce8-4129-4fe0-bf8c-8057801e2dd8 has been disabled on the PIM side. Note that you can get catalogs status with the GET /api/rest/v1/catalogs endpoint."